examples/particle-gibbs/script.jl

```diff
@@ -137,3 +134,38 @@ plot(
     ylabel="Update rate",
 )
 hline!([1 - 1 / Nₚ]; label="N: $(Nₚ)")
+
+# Let's see if ancestor sampling can help with the degeneracy problem. We use the same number of particles, but replace the sampler with PGAS. 
+# To use this sampler we need to define the transition and observation densities as well as the initial distribution in the following way:
+AdvancedPS.initialization(model::NonLinearTimeSeries) = f₀(model)
+AdvancedPS.transition(model::NonLinearTimeSeries, state, step) = f(model, state, step)
+function AdvancedPS.observation(model::NonLinearTimeSeries, state, step)
+    return logpdf(g(model, state, step), y[step])
+end
+AdvancedPS.isdone(::NonLinearTimeSeries, step) = step > Tₘ
+
+# We can now sample from the model using the PGAS sampler and collect the trajectories.
+pg = AdvancedPS.PGAS(Nₚ)
+chains = sample(model, pg, Nₛ);
+trajectories = map(chains) do sample
+    replay(sample.trajectory)
+end
```

Comment:
This is already done earlier in the code and `PGAS` stores the individual trajectories, no need to replay 
```suggestion
```